src/Interfaces.ts

```diff
@@ -17,10 +16,30 @@ export function isClaim(object: any): object is Claim {
     object.signature &&
     object.type &&
     object.attributes &&
-    object.dateCreated instanceof Date
+    object.created instanceof Date
   )
 }
 
+// WARNING: This MUST account for ALL of the attributes in a Claim, except for id, @context, and signature.
+// Otherwise those attributes without context will be left out of the canonized/signed claim.
+// Refer to https://www.w3.org/2018/jsonld-cg-reports/json-ld/#the-context
+export function getClaimContext(): any {
```

Comment:
We could declare this as a `const` instead of a function and mark all fields as `readonly`.

```ts
interface ClaimContext {
    readonly '@context': {
      readonly [key: string]: string
    }
}

const ClaimContext: ClaimContext = {
    '@context': {
        attributes: 'http://schema.org/CreativeWork',
        // etc
    }
}
```

Attempting to mutate ClaimContext in any way will result in an error:

```
index.ts:14:1 - error TS2542: Index signature in type '{ readonly [key: string]: string; }' only permits reading.
```